core/commonMain/src/ArgParser.kt

```diff
@@ -51,14 +51,16 @@ interface ArgumentValueDelegate<T> {
      *
      * @see CLIEntity.value
      */
-    // TODO: decide on the exception type
-    // TODO: decide whether 'set' is possible before calling 'parse'
```

Comment:
After we decided that, we need to reflect it in the property contract in the documentation